src/plugins/content_management/public/components/section_render.tsx

```diff
@@ -79,5 +79,9 @@ export const SectionRender = (props: Props) => {
     return <CardSection {...props} />;
   }
 
+  if (props.section.kind === 'custom') {
```

Comment:
Btw, I added the render for custom section in this PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7201/files#diff-7b2f3469233ec1de16d487a2ab3375d0454cedff616315f8809c91e697495d45

Reply:
Thanks, I will rebase that.